pekko/src/main/scala/org/scanamo/ScanamoPekko.scala

```diff
@@ -16,74 +16,43 @@
 
 package org.scanamo
 
-import cats.{ ~>, Monad }
+import cats.Monad
 import org.apache.pekko.NotUsed
 import org.apache.pekko.actor.ClassicActorSystemProvider
 import org.apache.pekko.stream.scaladsl.{ Sink, Source }
-import org.scanamo.ops.PekkoInterpreter.Pekko
-import org.scanamo.ops.{ PekkoInterpreter, ScanamoOps, ScanamoOpsT }
+import org.scanamo.PekkoInstances.*
+import org.scanamo.ops.{ PekkoInterpreter, ScanamoOps }
 import software.amazon.awssdk.services.dynamodb.DynamoDbAsyncClient
-import software.amazon.awssdk.services.dynamodb.model.*
 
 import scala.concurrent.Future
 
-/** Provides the same interface as [[org.scanamo.Scanamo]], except that it requires an
-  * [[https://github.com/apache/incubator-pekko-connectors Pekko connectors]] client, a
-  * [[org.scanamo.ops.retrypolicy.RetryPolicy]] and a predicate for which [[scala.Throwable]]s should be retried.
-  * `retryPolicy` defaults to [[org.scanamo.ops.retrypolicy.RetryPolicy.max]] with maximum 3 retries if not explicitly
-  * provided. `isRetryable` defaults to retry the most common retryable Dynamo exceptions. Moreover, the interface
-  * returns either a [[scala.concurrent.Future]] or [[org.apache.pekko.stream.scaladsl.Source]] based on the kind of
-  * execution used.
+/** The interface returns either a [[scala.concurrent.Future]] or [[org.apache.pekko.stream.scaladsl.Source]] based on
+  * the kind of execution used.
```

Comment:
The comments about retry logic are obsolete since https://github.com/scanamo/scanamo/pull/825 removed Scanamo's superfluous retry logic in August 2020, as per https://github.com/scanamo/scanamo/issues/819.